src/SixLabors.Fonts/TextLayout.cs

```diff
@@ -391,12 +391,23 @@ private static IEnumerable<GlyphLayout> LayoutLineHorizontal(
             TextLine.GlyphLayoutData data = textLine[i];
             if (data.IsNewLine)
             {
+                glyphs.Add(new GlyphLayout(
+                    new Glyph(data.Metrics[0], data.PointSize),
+                    boxLocation,
+                    penLocation,
+                    Vector2.Zero,
+                    data.ScaledAdvance,
+                    yLineAdvance,
+                    GlyphLayoutMode.Horizontal,
+                    true,
+                    data.GraphemeIndex,
+                    data.StringIndex));
+
                 penLocation.X = originX;
                 penLocation.Y += yLineAdvance;
-
                 boxLocation.X = originX;
                 boxLocation.Y += advanceY;
-                continue;
+                goto end;
```

Comment:
surely this would be the same as
```suggestion
                return glyphs;
```
for this particular case? (there's a good chance i'm missing something that make the goto worth it)

Reply:
Yeah, I shouldn't code late at night.